ci-operator/step-registry/cloud-bulldozer/e2e-benchmarking/virt-density/cloud-bulldozer-e2e-benchmarking-virt-density-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+pushd /tmp
+
+if [[ "$JOB_TYPE" == "presubmit" ]] && [[ "$REPO_OWNER" = "cloud-bulldozer" ]] && [[ "$REPO_NAME" = "e2e-benchmarking" ]]; then
+    if [ ${BAREMETAL} == "true" ]; then
```

Comment:
Aren't you using public VLAN deployments?
If that's the case, you shouldn't need this IF_BAREMETAL snippet

Reply:
Are you suggesting that we don't need a ssh proxy and then to execute commands from the bastion host if we select a public VLAN?